src/index.ts

```diff
@@ -36,7 +36,7 @@ const NanoClamp = ({
   lines = 3,
   text,
   ...props
-}: Props): JSX.Element | null => {
+}: Props): React.ReactElement | null => {
```

Comment:
**thought:** I think we can do without this annotated return type and just infer it:

```suggestion
}: Props) => {
```

Reply:
Done

Reply:
I just saw in the other PR that your lint configuration requires all functions to have a return type.
https://github.com/microlinkhq/nanoclamp/actions/runs/8409288351/job/23027852174?pr=29
So I will add the return type again.